Source/JavaScriptCore/API/tests/testapi.c

```diff
@@ -1195,13 +1195,13 @@ static void checkJSStringOOBUTF8(void)
     const size_t sourceCStringSize = 200;
     const size_t outCStringSize = 10;
 
-    char sourceCString[sourceCStringSize];
-    memset(sourceCString, 0, sizeof(sourceCString));
+    char *sourceCString = (char*)malloc(sourceCStringSize);
```

Comment:
We should put `*` to `char` side since this is `C` file and it is C pointer (not Objective-C pointers).